package.json

```diff
@@ -101,6 +101,7 @@
     "@babel/core": "patch:@babel/core@npm%3A7.21.5#./.yarn/patches/@babel-core-npm-7.21.5-c72c337956.patch",
     "@babel/runtime": "patch:@babel/runtime@npm%3A7.18.9#./.yarn/patches/@babel-runtime-npm-7.18.9-28ca6b5f61.patch",
     "@metamask/approval-controller": "^3.3.0",
+    "@metamask/message-manager": "^7.0.2",
```

Comment:
We don't really need a resolution when all other deps are on v7 do we? 🤔 

Should be enough with a regular patch bump.

Reply:
Good point, removed the resolution but updated the lock file.

Reply:
Looks like you may want to run `yarn dedupe` too